doc/src/manual/functions.md

```diff
@@ -66,16 +66,23 @@ Python, Ruby and Perl, among other dynamic languages.
 ## Argument-type declarations
 
 You can declare the types of function arguments by appending `::TypeName` to the argument name, as usual for [Type Declarations](@ref) in Julia.
-For example, the following function computes [Fibonacci numbers](https://en.wikipedia.org/wiki/Fibonacci_number) recursively:
+For example, the following function computes [Fibonacci numbers](https://en.wikipedia.org/wiki/Fibonacci_number) and returns the `n`th fibonacci number:
 ```
-fib(n::Integer) = n ≤ 2 ? one(n) : fib(n-1) + fib(n-2)
```

Comment:
Well, technically, this should really be as follows, but I suppose that for simplicity (one-liner and not using two ternary operators) the original is acceptable.
```julia
fib(n::Integer) = n < 1 ? zero(n) : n ≤ 2 ? one(n) : fib(n-1) + fib(n-2)
```

Reply:
@andrewjradcliffe you're right. But that part of the docs is introducing functions, so it should be **DEAD** simple, so anyone can follow through.